app/src/organisms/MapViewButton/index.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+
+import {
+  Btn,
+  Flex,
+  Icon,
+  ALIGN_CENTER,
+  COLORS,
+  DIRECTION_ROW,
+  SPACING,
+  TYPOGRAPHY,
+} from '@opentrons/components'
+import { StyledText } from '../../atoms/text'
+
+export function MapViewButton(
+  props: React.HTMLProps<HTMLButtonElement>
+): JSX.Element {
+  const { t } = useTranslation('protocol_setup')
+  return (
+    <Btn
+      position="fixed"
+      bottom="1.5rem"
+      right="1.5rem"
+      backgroundColor={COLORS.highlightPurple_one}
+      borderRadius="2.6875rem"
+      color={COLORS.white}
+      padding={`0.75rem ${SPACING.spacing5}`}
+      onClick={props.onClick}
+      width="15.25rem"
+    >
+      <Flex
+        alignItems={ALIGN_CENTER}
+        flexDirection={DIRECTION_ROW}
+        gridGap={SPACING.spacing3}
+      >
+        <Icon name="deck-map" width={SPACING.spacing7} color={COLORS.white} />
```

Comment:
basically spacing is for padding/margin/gridGap. for icon size, we can use `SIZE_number`.
but we don't have 3rem so need to hard-code...
```suggestion
        <Icon name="deck-map" width="3.75rem" height="3rem" color={COLORS.white} />
```

![Screenshot 2023-04-26 at 2 24 04 PM](https://user-images.githubusercontent.com/474225/234668356-d129f7f8-ee65-4a66-af95-25fdfad1082b.png)